core/src/main/scala/com/mesosphere/usi/core/CachedPendingLaunch.scala

```diff
@@ -1,6 +1,11 @@
 package com.mesosphere.usi.core
 import com.mesosphere.usi.core.models.{Goal, PodId, PodRecord}
 
+case class UpdateResult(
+    newCachedPendingLaunch: CachedPendingLaunch,
```

Comment:
I still have issues with naming here. If everything that `UpdateResult` has is _new_ then maybe drop it? How about:
```scala
pendingLaunch: CachedPendingLaunch,
ReadyToLaunch: Set[PodId],
alreadyLaunched: Set[PodId]
```